app/javascript/utils/reactTableUtils.js

```diff
@@ -14,3 +14,10 @@ const date = (cell) => {
 export const cellFormat = {
   date
 }
+
+export const filterMethod = (filter, row) => {
+  return (
+    // do case insensitive RegExp match instead of the default "startsWith"
+    row[filter.id].match(new RegExp(filter.value, 'ig'))
```

Comment:
I was a ,little confused at this code at first, I thought maybe `row` was actually a map of rowids to rows and we were getting the specific row by the row id, but I think what's actually happening is row is a map of column id to cells?

I think this would be clearer for me if we moved filter.id to a const like `columnId`

Reply:
went ahead and updated this